tests/API1/testcolorparameter.py

```diff
@@ -37,3 +37,8 @@ class Q(param.Parameterized):
         Q.q = '#fff'
         self.assertEqual(Q.q, '#fff')
 
+    def test_valid_named_color(self):
+        class Q(param.Parameterized):
+            q = param.Color(allow_named=True)
+        Q.q = 'indianred'
+        self.assertEqual(Q.q, 'indianred')
```

Comment:
Would be good to test for an invalid color name when `allow_named=True`, as well.